### PR TITLE
add tuto and example about results of algorithms

### PR DESCRIPTION
**Pull request by geoffreyp**

### CODE REVIEW

File: docs/tuto.rst [hunk 1]
```diff
@@ -144,3 +144,46 @@ If you want to save all non-dominated solutions (attribute :code:`self.ep` in th
             save_population(file_name=filename, population=moead.ep)
     
     moead.run(checkpoint=checkpt)
+
+
+Extract and plot the Pareto front
+--------------------------------------------------------------------
+
+The method `run()` of each algorithm return a list of :class:`moead_framework.solution.one_dimension_solution.OneDimensionSolution`.
```
Comment (sjvrijn) [inline docs/tuto.rst:152]: grammar: returns

File: docs/tuto.rst [hunk 2]
```diff
@@ -144,3 +144,46 @@ If you want to save all non-dominated solutions (attribute :code:`self.ep` in th
...
+
+    moead = Moead(
+              problem=rmnk,
+              max_evaluation = number_of_evaluations,
+              number_of_weight_neighborhood=number_of_weight_neighborhood,
+              weight_file=weight_file,
+              aggregation_function=Tchebycheff
+              )
+
+    list_of_solution = moead.run(checkpoint=checkpt)
```
Comment (sjvrijn) [inline docs/tuto.rst:164]: grammatically correct would be `list_of_solutions`, although just `solutions` would also be fine

**Review by sjvrijn — CHANGES_REQUESTED** [review]
Great! Thanks for adding these examples.
All fine by me after the two spelling updates.

**Comment by geoffreyp** [comment]
Thank you, it's done in c39ca19

**Review by sjvrijn — APPROVED** [review]

**Pull request merged**